### DISCUSSION TITLE
Spanish translators discussion group

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/Web-Dev-For-Beginners#537] [Translations]
Let's sync here on translations in Spanish!

**Comment by Lakshman-Ch-maker** [comment]
Please, unsubscribe me from your mailing list.

**Reply by Groangfr** [reply]
Hello

**Reply by Groangfr** [reply]
Helo

**Comment by JoelBuenrostro** [comment]
Hi, what is the status of the Spanish translation of this content? I can see it in another place?
